test/core/TakerOperations.t.sol

```diff
@@ -297,14 +280,15 @@ contract TakerOperationsTest is MangroveTest {
 
     expectFrom($(mgv));
     emit OfferFail($(base), $(quote), ofr, $(this), 1 ether, 1 ether, "mgv/makerTransferFail");
-    (uint successes, uint takerGot, uint takerGave,,) =
-      testMgv.snipesInTest($(base), $(quote), wrap_dynamic([ofr, uint(Tick.unwrap(offerTick)), 1 ether, 100_000]), true);
+    vm.expectEmit(true, false, false, false, $(mgv));
+    emit Credit($(refusemkr), 0); // credited amount should be "mkr_provision - penalty"; Not including this in the emit assertion, as penalty is difficult to maintain.
```

Comment:
I don't understand this comment

Reply:
Ideally, we'd include the credited value in the emitted event. However, the penalty isn't something we can calculate (since it depends on actual gas usage) and I think it's likely to change over time, so putting in a constant would require maintenance.

Do you think we should something different?

Reply:
Addressed in [a21d89d](https://github.com/mangrovedao/mangrove-core/pull/454/commits/a21d89dbc0f556eda8cdea09e69e21f89da9b907)